.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+ version: 2
+
+ build:
+   os: "ubuntu-24.04"
+   tools:
+     python: "3.13"
+   # We recommend using a requirements file for reproducible builds.
+   # This is just a quick example to get started.
+   # https://docs.readthedocs.io/page/guides/reproducible-builds.html
+   jobs:
+    create_environment:
+      - asdf plugin add uv
+      - asdf install uv latest
+      - asdf global uv latest
+      - VENV_EXTRAS="--all-extras" VENV_DIR="$READTHEDOCS_VIRTUALENV_PATH" make venv
```

Comment:
I had to look up https://docs.readthedocs.com/platform/stable/reference/environment-variables.html#envvar-READTHEDOCS_VIRTUALENV_PATH. Not sure if it's worth adding it as a comment in similar fashion to the existing comment above.